homeassistant/components/jewish_calendar/template_functions.py

```diff
@@ -0,0 +1,85 @@
+"""Template functions to return HDate instances & other information."""
+
+
+from __future__ import annotations
+
+import datetime as dt
+import logging
+from typing import cast
+
+from hdate import HDate, Zmanim
+
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import template
+from homeassistant.helpers.typing import ConfigType
+import homeassistant.util.dt as dt_util
+
+DOMAIN = "jewish_calendar"
+
+LOGGER = logging.getLogger(__name__)
+
+
+class JewishCalendarTemplates:
+    """Registers services for the jewish_calendar integration."""
+
+    def __init__(self, hass: HomeAssistant, config: ConfigType) -> None:
+        """Register the services."""
+        data = hass.data[DOMAIN]
+
+        self._location = data["location"]
+        self._hebrew = data["language"] == "hebrew"
+        self._candle_lighting_offset = data["candle_lighting_offset"]
+        self._havdalah_offset = data["havdalah_offset"]
+        self._diaspora = data["diaspora"]
+
+        template.register_function(
+            hass,
+            "get_hebrew_date",
```

Comment:
Maybe the function name as a filter should be "hebrew_date"? It matches better the Jinja semantics.